docs/docs/concepts/email-sms.md

```diff
@@ -201,6 +261,45 @@ the following pattern: `email.body*`. You can also see that the `Identity` of
 the user is available in all templates, and that you can use Sprig functions
 also in the nested templates.
 
+### Nested templates with remote templates
+
+When remote templates are used in Kratos, the dynamics of loading nested
+templates change. The templates cannot reference templates outside itself as
+with templates loaded from a singular directory.
+
+The template will need to contain the nested templates in the same file. See
+below for an example. In this template we even wrap the main content in a define
+block called "base". We can then call the "base" define block by using the
+`template_root` configuration.
```

Comment:
So `template_root` defines the primary template "id" that is used to render the email? If so, maybe we could change it to `use_id` or `use`? Alternatively we could also just expect this to be e.g. `body` or `subject` which would match the template target and omit this config key?

Reply:
Yeah, I agree that we can just always expect a `body` key defined and a `subject`. But this would mean you will always bundle everything in a singular template with the translations. Is that okay? Would we then also expect plaintext and html to be inside the same gotmpl?

e.g.

```
{{ define "body.html" }}
 <h1>an html body</h1>
{{- end -}}

{{ define "subject" }}

{{- end -}}

{{ define "body.plaintext" }}
a plaintext body
{{- end -}}

{{ define "en.US" }}
language specific template
{{- end -}}
```

Reply:
Then in the config we only have

```yaml

courier:
  templates:
    recovery:
      invalid: "https://remote-template/gotmpl"
...
```